cmd/dbseed/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/thrasher-corp/gocryptotrader/cmd/dbseed
+
+go 1.14
+
+require (
+	github.com/thrasher-corp/gocryptotrader v0.0.0-20200724031809-14c72c9c6b45
+	github.com/thrasher-corp/sqlboiler v1.0.1-0.20191001234224-71e17f37a85e
+	github.com/urfave/cli/v2 v2.2.0
+)
+
+replace github.com/thrasher-corp/gocryptotrader => ./../../
```

Comment:
I will remove this being a standalone go module it was originally just intended to keep the cli v2 module out of the main gct module since we are still on v2 but that has some limitations like not being able to run it from outside the path (that only impacts me apparently 😆)

and in the end there isn't many breaking changes from v1 to v2 that gctcli can easily be upgraded at some point